packages/core/src/internal/motion/index.ts

```diff
@@ -14,6 +14,7 @@ export {
   AnimationAccordionPanelOpenConfig,
   AnimationAccordionPanelOpenName,
 } from './animations/cds-accordion-panel-open.js';
+export { AnimationTreeItemExpandConfig, AnimationTreeItemExpandName } from './animations/cds-tree-item-expand.js';
```

Comment:
This is fine here right now. But I wanted to give a heads up to @coryrylan because I think this file goes away in the lid 2 upgrade.

Reply:
Yeah this is a nested barrel which rollup does not allow due to treeshaking issues. This would be exported in the `/internal/index.ts` file instead

Reply:
I won't move the existing exports but will add a comment that we should be moving them to `/internal/index.ts`.

Reply:
This is probably something you'll need to do with the Lit 2 rebase...

Reply:
addressed with the rebase 👍